recipes/opencv/2.x/test_package/CMakeLists.txt

```
@@ -0,0 +1,10 @@
+cmake_minimum_required(VERSION 3.1)
+project(test_package)
+
+include(${CMAKE_BINARY_DIR}/conanbuildinfo.cmake)
+conan_basic_setup()
+
+find_package(OpenCV REQUIRED CONFIG)
```

```suggestion
find_package(OpenCV REQUIRED imgproc highgui objdetect CONFIG)
```

see https://github.com/conan-io/conan-center-index/pull/2743/files#r512513403

---

Done